frameworks/helloworld/tests/scale/test_scale.py

```diff
@@ -18,41 +22,97 @@ def test_soak_load(service_name,
         service_name: name of the service to install as
         scenario: yaml scenario to run helloworld with (normal, crashloop) are added for this case
     """
-    # Note service-names *cannot* have underscores in them.
-    soak_service_name = ("{}-{}-soak".format(service_name, scenario)).replace("_", "-")
-    # service-names can have '/'s in them but service account names cannot, sanitize here.
-    service_account_name = soak_service_name.replace("/", "__")
-    security_info = _create_service_account(service_account_name)
-    _install_service(soak_service_name,
-                     scenario,
-                     security_info)
+    soak_service_name = "{}-{}-soak".format(service_name, scenario)
+    _launch_load(soak_service_name, scenario)
 
 
 @pytest.mark.scale
 def test_scaling_load(service_name,
+                      scenario,
                       service_count,
-                      scenario) -> None:
+                      min_index,
+                      max_index,
+                      batch_size) -> None:
+    """Launch a load test scenario in parallel if needed.
+    This does not verify the results of the test, but does
+    ensure the instances were created.
+
+    Args:
+        service_name: name of the service to install as
+        scenario: yaml scenario to run helloworld with (normal, crashloop) are added for this case
+        service_count: number of helloworld services to install
+        min_index: minimum index to begin suffixes at
+        max_index: maximum index to end suffixes at
+        batch_size: batch size to deploy instances in
+    """
+    scale_service_name = "{}-{}".format(service_name, scenario)
+    deployment_list = []
+    if min_index == -1 or max_index == -1:
+        deployment_list = ["{}-{}".format(scale_service_name, index) for index in range(0, int(service_count))]
+        min_index = 0
+        max_index = service_count
+    else:
+        deployment_list = ["{}-{}".format(scale_service_name, index) for index in range(min_index, max_index)]
+
+    sdk_security.install_enterprise_cli()
+
+    current = 0
```

Comment:
Does this code support the following use case:

1. Run the script once to launch 25 instances (0 to 24) of hw.
2. Script exits (may be some other person wants to takes over at a later point in time)
3. Run the script again to launch instances (25 to 49) of hw.

Reply:
Yes.